src/pages/[platform]/build-a-backend/storage/download-files/index.mdx

```diff
@@ -151,6 +151,36 @@ RxAmplify.Storage.getUrl(StoragePath.fromString("public/example")).subscribe(
 
 </Block>
 </BlockSwitcher>
+
+### Check the existence of a file
+
+When creating a downloadable URL, you can choose to check if the file exists by setting `validateObjectExistence` to
+`true` in `AWSS3StorageGetPresignedUrlOptions`. If the file is inaccessible or does not exist, a `StorageException` is thrown.
+This allows you to check if an object exists during generating the presigned URL, which you can then use to download
+that object.
+
+
+```kotlin
```

Comment:
Should we include a block switcher to show a code example for java, kotlin coroutine, and rxjava?

Reply:
good suggestion, block switcher added.